bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/ExtensionsPage.java

```diff
@@ -357,6 +384,8 @@ public void renderHead(IHeaderResponse response) {
         super.renderHead(response);
 
         response.render(CssHeaderItem.forReference(OpenWebIconsCssReference.instance()));
+		response.render(CssHeaderItem.forReference(new WebjarsCssResourceReference("font-awesome/current/css/font-awesome.css")));
```

Comment:
You could use `FontAwesomeCssResourceReference` (or something similar). We have such resref in -extensions already.